lnbits/templates/base.html

```diff
@@ -89,7 +89,7 @@
             >
               {% if LNBITS_SERVICE_FEE_MAX > 0 %}
               <span
-                v-text='$t("service_fee_max_badge", { amount: "{{ LNBITS_SERVICE_FEE }}", max: "{{ LNBITS_SERVICE_FEE_MAX }}", denom: "{{ LNBITS_DENOMINATION }}"})'
+                v-text='$t("service_fee_max_badge", { amount: "{{ LNBITS_SERVICE_FEE }}", max: "{{ LNBITS_SERVICE_FEE_MAX }} {{ LNBITS_DENOMINATION }}"})'
```

Comment:
i dont understand this, change its equivalent, if you combine the 2 variable into 1 replacement for the i18n string. you have to remove it from the i18n labels, but why?